cmd/fortifyExecuteScan.go

```diff
@@ -123,6 +126,14 @@ func determineArtifact(config fortifyExecuteScanOptions, utils fortifyUtils) (ve
 func runFortifyScan(config fortifyExecuteScanOptions, sys fortify.System, utils fortifyUtils, telemetryData *telemetry.CustomData, influx *fortifyExecuteScanInflux, auditStatus map[string]string) ([]piperutils.Path, error) {
 	var reports []piperutils.Path
 	log.Entry().Debugf("Running Fortify scan against SSC at %v", config.ServerURL)
+	_, err := execInPath("fortifyupdate")
+	if err != nil {
+		return reports, fmt.Errorf("ERROR , command not found: fortifyupdate")
```

Comment:
I propose to append a sentence like the following to the error message:
"Please configure a supported docker image or install Fortify SCA on the system."

Reply:
Thanks. Let's really do "append" and not "replace", though. Altogether:

"fortifyupdate not found. Please..."